job/task.sh

```diff
@@ -50,9 +50,25 @@ always() {
 }
 <% } %>
 
+init_integrations() {
```

Comment:
- can we move this to a file called `integrations_init.sh` and inject it before we run tasks
- same for cleanup. the file should be `integrations_cleanup.sh` and should be injected after everything is completed